src/main/kotlin/edu/carroll/polyominoes/service/game/GameService.kt

```diff
@@ -0,0 +1,26 @@
+package edu.carroll.polyominoes.service.game
+
+import edu.carroll.polyominoes.web.rest.ajax.GameStats
+
+interface GameService {
+
+    /**
+     *  When given a username and GameStats saves them to the corresponding in the database for the given user.
```

Comment:
Either use uppercase for both parameter names, or lowercase both. username is a String, and GameStats is a type. Consistency is key!
```
When given a username and gamesStats saves them to the corresponding in the database for the given user.
```
Or
```
When given a String and GameStats saves them to the corresponding in the database for the given user.
```
(I think the former is more descriptive...)